unit_test/hip/Test_HIP_Blas1_team_axpy.cpp

```diff
@@ -0,0 +1,4 @@
+#include<Test_HIP.hpp>
+#ifdef KOKKOS_ENABLE_CUDA_LAMBDA
```

Comment:
@lucbv probably preemptive comment since this is WIP, but the CUDA guard should be removed (there is no lambda guard for HIP in Kokkos)